src/coreclr/src/inc/crosscomp.h

```diff
@@ -113,8 +113,7 @@ typedef struct DECLSPEC_ALIGN(8) _T_CONTEXT {
 // each frame function.
 //
 
-#ifndef TARGET_UNIX
-#ifdef HOST_X86
+#if defined(HOST_WINDOWS) && defined(HOST_X86)
 typedef struct _RUNTIME_FUNCTION {
```

Comment:
We have the proper cross-targeting `T_XXX` definition below. We should not need these for anything.

All code using these definitions should be unreachable. Is the whole purpose of these definitions to just make some unreachable code compile?

Reply:
>  Is the whole purpose of these definitions to just make some unreachable code compile?

These types seem to be defined because they are missing from the host compiler's types.  

I also suspect the original code was covering over places where we should have been using T_RUNTIME_FUNCTION